packages/lesswrong/server/notificationTypesServer.tsx

```diff
@@ -598,3 +599,26 @@ export const NewSubforumMemberNotification = serverRegisterNotificationType({
     </div>
   },
 });
+
+export const NewMentionNotification = serverRegisterNotificationType({
+  name: "newMention",
+  emailSubject: async ({ user, notifications }: {user: DbUser, notifications: DbNotification[]}) => {
+    const summary = await getDocumentSummary(notifications[0].documentId as NotificationDocument, notifications[0].documentType);
```

Comment:
The cast seems wrong, I think it should be on `documentType`?  `documentId` is the actual `_id` of the document being referenced by the notification

Reply:
whelp 🙈. nice catch!

Reply:
I suppose I'd ideally propagate the type to the DB (update allowed values, etc) but when I looked at it - it felt like that refactoring was expanding too much in scope 🤔